packages/release-please/src/release-please.ts

```diff
@@ -520,6 +522,49 @@ const handler = (app: Probot) => {
         octokit,
         {logger, skipPullRequest: branchConfiguration.onDemand}
       );
+      if (branchConfiguration.tagPullRequestNumber) {
+        context.payload.commits.forEach(c => commits.add(c.id));
+      }
+    }
+    const commitsAsList = [...commits];
+    if (commitsAsList.length !== 1) {
+      logger.info(
+        'The number of commits is not 1. Did the PR get squash-merged properly? Commits:',
+        commitsAsList
+      );
+      return;
+    } else {
```

Comment:
This `else` shouldn't be necessary as the previous conditional is a guard that stops execution.